From dbc6c842e390ddfa83a5a68d72df3ba5b050e12f Mon Sep 17 00:00:00 2001
From: Ivan Chavero <ichavero@redhat.com>
Date: Thu, 7 May 2015 22:58:03 -0600
Subject: [PATCH] Fix mongodb configuration

RHEL OSP 7 now uses /etc/mongod.conf instead of /etc/mongodb.conf

Warning: this patch must not be backported to Juno. RHEL OSP 6
still uses /etc/mongodb.conf

Change-Id: Idc12c3fb2c455879d0325b5126e8c04b6971cf50
---
 packstack/puppet/templates/mongodb.pp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/packstack/puppet/templates/mongodb.pp b/packstack/puppet/templates/mongodb.pp
index afaeb68..3b1a83a 100644
--- a/packstack/puppet/templates/mongodb.pp
+++ b/packstack/puppet/templates/mongodb.pp
@@ -1,10 +1,9 @@
 $mongodb_host = hiera('CONFIG_MONGODB_HOST')
-$use_epel = hiera('CONFIG_USE_EPEL')
 
 # The MongoDB config files differ between versions
 if (($::operatingsystem == 'fedora' and versioncmp($::operatingsystemrelease, '22') >= 0)
     or
-    ($::operatingsystem != 'fedora' and versioncmp($::operatingsystemrelease, '7.0') >= 0 and $use_epel == 'y')
+    ($::operatingsystem != 'fedora' and versioncmp($::operatingsystemrelease, '7.0') >= 0)
    ){
   $config_file = '/etc/mongod.conf'
 } else {
-- 
1.9.3

